samples/matter/window_covering/sample.yaml

```diff
@@ -0,0 +1,35 @@
+sample:
+  name: Matter Window Covering
+  description: Matter Window Covering example
+tests:
+  matter.lock.debug:
+    build_only: true
+    platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp
+    integration_platforms:
+      - nrf52840dk_nrf52840
+      - nrf5340dk_nrf5340_cpuapp
+    tags: ci_build
+  matter.lock.release:
+    build_only: true
+    platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp
+    integration_platforms:
+      - nrf52840dk_nrf52840
+      - nrf5340dk_nrf5340_cpuapp
+    tags: ci_build
+    extra_args: CONF_FILE=prj_release.conf
+  matter.lock.no_dfu:
```

Comment:
I Think that better option is to have matter.window_covering.both_dfu or something simmilar

Reply:
The whole sample.yaml file was revised.